src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -68,7 +68,16 @@
       (compojure/GET "/services" []
         (let [statuses (call-status-fns status-fns-atom)]
           {:status 200
-           :body {"services" statuses}})))))
+           :body statuses}))
+       (compojure/GET "/services/:service-name" [service-name]
+         (if-let [status-info (get (deref status-fns-atom) service-name)]
+           (let [status (call-latest-status-fn-for-service status-info)]
+             {:status 200
+              :body (assoc status :service-name service-name)})
+           {:status 404
+            :body {:error {:type ::service-not-found
```

Comment:
Hmm, yeah, I can see something like that being useful in general, but ...

In this specific case, a user doesn't care about the status service at all - hopefully, they don't even know it exists.  They'll be querying for the status of an individual service they care about, e.g. Puppet Server, so anything about `status-service` in the response is probably just noise (at least in this use-case).

I feel like we're treading on :bike: territory here ... I vote for just `:service-not-found` but since we don't have a hard requirement one way or the other at this point, I'll be fine with whatever you want to go with.